src/Registrar.php

```diff
@@ -54,11 +54,9 @@ public function getInstance($class)
 
     public function register()
     {
-        foreach ($this->di as $classes) {
-            foreach ($classes as $instance) {
-                if ($instance instanceof \Dxw\Iguana\Registerable) {
-                    $instance->register();
-                }
+        foreach ($this->di[$this->namespace] as $instance) {
```

Comment:
Should `$namespace` be declared as a (private?) property at the top of this class?

Reply:
Yes. I made it `protected` as all the other instance variables are `protected`.